model/perf.go

```diff
@@ -637,3 +638,151 @@ func (r *PerformanceResults) FindOutdatedRollups(ctx context.Context, name strin
 
 	return errors.WithStack(it.Close(ctx))
 }
+
+func GetTimeSeriesIds(ctx context.Context, db *mongo.Database) (*mongo.Cursor, error) {
+	pipeline := bson.A{
+		bson.M{
+			"$match": bson.M{
+				"info.order": bson.M{
```

Comment:
are these field names in pacakge level constants (bsonutil.MustHaveKey values)?

Reply:
I can onlt find `bsonutil.MustHaveTag` --> where is `MustHaveKey`?

Reply:
sorry, I remembered incorrectly, MustHaveTag is the one we want.

Reply:
bump

Reply:
This seems much more complicated than what I need here.